ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3593,8 +3800,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnecting-network
-- as: vsphere-upi-encrypt-f14
-  cron: 19 17 2,17 * *
+- as: vsphere-upi-disconnecting-network-f28-destructive
+  cron: 57 8 28 * *
+  steps:
+    cluster_profile: vsphere-connected
+    env:
+      E2E_RUN_TAGS: '@amd64 and @vsphere-upi and @disconnected and @network-openshiftsdn
```

Comment:
by a chance, I saw `@network-openshiftsdn` is set here, do you mind to update it to `@network-ovnkubernetes` together with `vsphere-upi-disconnecting-network-f28` job.